src/resources/help/en_US/relnotes3.3.3.html

```diff
@@ -30,6 +30,7 @@
   <li>Added domain variable ordering to the metadata that comes out of the X Ray Image Query Conduit outputs.</li>
   <li>Changed the metadata coming out of the X Ray Image Query Conduit outputs to use pot_hole_case as opposed to camelCase.</li>
   <li>Built and installed an x86_64 toss4 version of VisIt on the LLNL systems.</li>
+  <li>Updated the VTKm library to version 1.9.0, which has added support for AMD GPUs.</li>
```

Comment:
For users, was there ever a release with an older `VTKm` library (as opposed to `VTKh`?)? I guess I had the understanding that this is an update from `VTKh` to `VTKm` and so was expecting...

```suggestion
  <li>Updated from VKTh version 0.8.0 to VTKm version 1.9.0 to add support for AMD GPUs.</li>
```

Reply:
Users were never aware of VTKh, so I think that may not mean anything to them. VTKh 0.8.0 used VTKm 1.7.0, so I think of it as upgrading VTKm from 1.7.0 to 1.9.0. I think I could make that more clear and I like your thought that we upgraded to add support for AMD GPUs, instead of that being just an artifact of the upgrade. I updated the release notes.